### ISSUE TITLE
performance with default ont model worse than clair2

### ISSUE THREAD

**Issue by KewinOgink** [HKU-BAL/Clair3#13]
Hi,

I tested a dataset with clair2 and the pretrained ont model [HG001,2,2HD,3,4
](https://github.com/HKU-BAL/Clair#pretrained-models) and compared it with the new clair3 default ont model ([HG001,2,4,5](https://github.com/HKU-BAL/Clair3/blob/main/docs/training_data.md#pre-trained-model))on the same sample.

<meta name="ProgId" content="Excel.Sheet">
<meta name="Generator" content="Microsoft Excel 15">
<link id="Main-File" rel="Main-File" href="file:///C:/Users/376946/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel="File-List" href="file:///C:/Users/376946/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>

Clair2 | Type | TRUTH.TOTAL | TRUTH.TP | TRUTH.FN | QUERY.TOTAL | QUERY.FP | QUERY.UNK | FP.gt | Recall | Precision | Frac_NA | .F1_Score
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
ont_DP10_AF0.7 | INDEL | 1681 | 817 | 864 | 34981 | 34129 | 0 | 7 | 0.48602 | 0.024356 | 0 | 0.046388
&nbsp; | SNP | 20759 | 19776 | 983 | 24744 | 3483 | 0 | 28 | 0.952647 | 0.859239 | 0 | 0.903535
Clair3 | &nbsp; | &nbsp; | &nbsp; | &nbsp; | &nbsp; | &nbsp; | &nbsp; | &nbsp; | &nbsp; | &nbsp; | &nbsp; | &nbsp;
ont_DP10_AF0.7_all_merge | INDEL | 1681 | 629 | 1052 | 33590 | 32932 | 0 | 9 | 0.374182 | 0.019589 | 0 | 0.037229
&nbsp; | SNP | 20759 | 19599 | 1160 | 24230 | 3112 | 0 | 27 | 0.944121 | 0.871564 | 0 | 0.906393

The precision and f1 of clair3 here is higher on SNPs, but the SNP recall and overall INDEL score is worse. Is this due to the training data difference? I used the same (default) settings and data, but I noticed the training data for clair2 and 3 are slightly different?

**Comment by aquaskyline** [comment]
Hi KewinOgink,

Is it possible for you to share with me your dataset? It'd be the fastest way for me to find out why. Please drop me an email at rbluo@cs.hku.hk for any links to your dataset.

**Comment by aquaskyline** [comment]
What's the Guppy version you were using for basecalling?

**Comment by KewinOgink** [comment]
the data was basecalled with metrichor

**Comment by aquaskyline** [comment]
Files well received and looking into it. Will turn back to you asap.

**Comment by aquaskyline** [comment]
Thanks again for sending over the data. We played with the data and have the following observations.
1. The yeast data was sequenced in 2017 and basecalled with Metrichor. The average base error rate is ~15.7%, which is about a time higher than the latest Guppy. The error patterns by Metrichor are also observably different from the latest Guppy. We found that many homozygous SNPs called by FreeBayes on the 197x Illumina data have been incorrectly called as heterozygotes by Clair3 on the 40x ONT Metrichor data. We believe that Clair3 cannot work with Metrichor data at the moment. 
2. We also found some indels called Clair3 been misclassified as FP because Freebayes has missed them on the Illumina data. This is a minor concern but had also been contributing to a lower count of TP and higher FP.

We suggest remain using Clair on Metrichor data at the moment. We will explore how much Metrichor data could be collected for Clair3 model training. If we found a Metrichor Clair3 model is at all possible, we will reply to this issue.
